### ISSUE TITLE
The filename, directory name, or volume label syntax is incorrect on Windows 10

### ISSUE THREAD

**Issue by its-me-mahmud** [VeryGoodOpenSource/dart_frog#35]
**Description**

The filename, directory name, or volume label syntax is incorrect.

```dart
dart_frog dev
```
After running the above command in cmd
```dart
✓ Running on http://localhost:8080 (3.3s)
The Dart VM service is listening on http://127.0.0.1:8181/45EwVYB4AYs=/

The Dart DevTools debugger and profiler is available at: http://127.0.0.1:8181/45EwVYB4AYs=/devtools/#/?uri=ws%3A%2F%2F127.0.0.1%3A8181%2F45EwVYB4AYs%3D%2Fws

.dart_frog/server.dart:7:8: Error: Error when reading '.dart_frog/..%0Doutes/index.dart': The filename, directory name, or volume label syntax is incorrect.

import '..\routes/index.dart' as routes_index;
       ^
.dart_frog/server.dart:29:29: Error: Undefined name 'onRequest'.
    ..all('/', routes_index.onRequest);
                            ^^^^^^^^^
```
**Environment**
```dart
flutter doctor -v
```
```dart
[√] Flutter (Channel stable, 3.0.0, on Microsoft Windows [Version 10.0.19043.1706],
    locale en-US)
    • Flutter version 3.0.0 at C:\src\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision ee4e09cce0 (2 weeks ago), 2022-05-09 16:45:18 -0700
    • Engine revision d1b9a6938a
    • Dart version 2.17.0
    • DevTools version 2.12.2

Checking Android licenses is taking an unexpectedly long time...[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\abdullah\AppData\Local\Android\sdk
    • Platform android-31, build-tools 30.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.0.4)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.0.32014.148
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[√] VS Code (version 1.67.2)
    • VS Code at C:\Users\abdullah\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.40.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version
      10.0.19043.1706]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 101.0.4951.67
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 100.0.1185.44

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Comment by Policy56** [comment]
'/routes/index.dart' is automatically rename to '\routes\index.dart' when run dart_frog dev

the 'r' of 'routes' has been omitted  

![image](https://user-images.githubusercontent.com/33663119/170200664-eb28aac6-b91a-49be-9e56-8acc2ee46008.png)

**Comment by felangel** [comment]
Thanks for opening an issue! I’ll take a closer look shortly 👍

**Comment by felangel** [comment]
This should be resolved by #48 and will be released early tomorrow. Thanks for your patience!

**Comment by felangel** [comment]
This should be fixed in dart_frog_cli v0.0.2-dev.2. You should just need to run `dart pub global activate dart_frog_cli` 🎉 
Let us know if you encounter any other issues and thanks for the patience!

**Comment by its-me-mahmud** [comment]
Now it's working. Thanks 💙
